tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -289,10 +318,14 @@ async function ensureCommerceMockWithCompassTestFixture(client, appName, scenari
   return mockHost;
 }
 
-async function ensureCommerceMockLocalTestFixture(mockNamespace, targetNamespace) {
+async function ensureCommerceMockLocalTestFixture(mockNamespace, targetNamespace, withCentralApplicationGateway=false) {
   
   await k8sApply(applicationObjs);
-  const mockHost = await provisionCommerceMockResources("commerce", mockNamespace, targetNamespace);
+  const mockHost = await provisionCommerceMockResources(
+      "commerce",
+      mockNamespace,
+      targetNamespace,
+      withCentralApplicationGateway ? lastorderCentralApplicationGatewayObjs : lastorderObjs);
```

Comment:
This would be simplified too... I guess.

Reply:
Yup